xivo_dao/alchemy/blocklist.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2025 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy import ForeignKey, select
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.associationproxy import association_proxy
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.schema import (
+    Column,
+    PrimaryKeyConstraint,
+)
+from sqlalchemy.sql import text
+from sqlalchemy.sql.schema import Index, UniqueConstraint
+from sqlalchemy.types import Text, String
+from sqlalchemy.orm import relationship
+
+from xivo_dao.alchemy.tenant import Tenant
+from xivo_dao.helpers.db_manager import Base
+from .userfeatures import UserFeatures
+
+
+class Blocklist(Base):
+    __tablename__ = 'blocklist'
+    __table_args__ = (
+        PrimaryKeyConstraint('uuid'),
+        Index('blocklist__idx__tenant_uuid', 'tenant_uuid'),
+    )
+
+    uuid = Column(UUID(as_uuid=False), server_default=text('uuid_generate_v4()'))
+    tenant_uuid = Column(
+        String(36),
+        ForeignKey(Tenant.uuid, ondelete='CASCADE'),
+        nullable=False,
+    )
+
+    tenant = relationship(Tenant, uselist=False)
+    numbers = relationship(lambda: BlocklistNumber)
```

Comment:
What's the use of the `lambda` here?

Reply:
forward(lazy) reference. The class is not yet defined,  but sqlalchemy supports lambdas for forward referencing models.